logic/src/commonTest/kotlin/com/wire/kalium/logic/feature/call/RejectCallUseCaseTest.kt

```diff
@@ -36,12 +41,22 @@ class RejectCallUseCaseTest {
             .whenInvokedWith(eq(conversationId))
             .thenDoNothing()
 
+        given(callRepository)
+            .suspendFunction(callRepository::updateCallStatusById)
+            .whenInvokedWith(eq(conversationId.toString()), eq(CallStatus.REJECTED))
+            .thenDoNothing()
+
         rejectCallUseCase.invoke(conversationId)
 
         verify(callManager)
             .suspendFunction(callManager::rejectCall)
             .with(eq(conversationId))
             .wasInvoked(once)
+
+        verify(callRepository)
+            .suspendFunction(callRepository::updateCallStatusById)
+            .with(eq(conversationId.toString()),  eq(CallStatus.REJECTED))
```

Comment:
```suggestion
            .with(eq(conversationId.toString()), eq(CallStatus.REJECTED))
```

Just some unnecessary extra space here